docs/management_api.md

```diff
@@ -30,7 +30,7 @@ Similar to the [Inference API](inference_api.md), the Management API provides a
 * response_timeout - If the model's backend worker doesn't respond with inference response within this timeout period, the worker will be deemed unresponsive and rebooted. The units is seconds. The default value is 120 seconds.
 
 ```bash
-curl -X POST "http://localhost:8081/models?url=https://<s3_path>/squeezenet_v1.1.mar"
+curl -X POST "http://localhost:8081/models?url=https://<s3_or_azureblob_path>/squeezenet_v1.1.mar"
```

Comment:
I believe this can be google cloud storage or any other cloud storage as well. Probably having a generic name as 'cloud_OR_noncloud-endpoint' and then providing explicit s3 or azure or gc example should help.

Reply:
fixed